### PR TITLE
modelo

### PR DESCRIPTION
**Pull request by LEMI92**
{
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:22791",
      "sslPort": 44390
    }
  },
  "profiles": {
    "Modelo": {
      "commandName": "Project",
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7049,http://localhost:5208",  // Cambio de punto y coma a coma
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}

### CODE REVIEW

**Comment by LEMI92** [comment]
{
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:22791",
      "sslPort": 44390
    }
  },
  "profiles": {
    "Modelo": {
      "commandName": "Project",
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7049;http://localhost:5208",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}

**Comment by LEMI92** [comment]
{
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:22791",
      "sslPort": 44390
    }
  },
  "profiles": {
    "Modelo": {
      "commandName": "Project",
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7049;http://localhost:5208",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}

**Comment by LEMI92** [comment]
chicos si no se ponen las pilas vamo  a reprobar.

**Comment by KarinaVillacorta21** [comment]
{
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:22791",
      "sslPort": 44390
    }
  },
  "profiles": {
    "Modelo": {
      "commandName": "Project",
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7049;http://localhost:5208",  // Cambio de punto y coma a coma
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}

**Comment by LEMI92** [comment]
😭😭😭😭😭😭😭😭😭😭😭 chicossssss me va a dar ansiedad!!!!!!

**Comment by LEMI92** [comment]
//este es el codigo correcto

{
"iisSettings": {
"windowsAuthentication": false,
"anonymousAuthentication": true,
"iisExpress": {
"applicationUrl": "http://localhost:22791/",
"sslPort": 44390
}
},
"profiles": {
"Modelo": {
"commandName": "Project",
"launchBrowser": true,
"applicationUrl": "[https://localhost:7049,http://localhost:5208](https://localhost:7049%2Chttp//localhost:5208)", // Cambio de punto y coma a coma
"environmentVariables": {
"ASPNETCORE_ENVIRONMENT": "Development"
}
},
"IIS Express": {
"commandName": "IISExpress",
"launchBrowser": true,
"environmentVariables": {
"ASPNETCORE_ENVIRONMENT": "Development"
}
}
}
}

**Comment by KarinaVillacorta21** [comment]
Chicos!?

**Review by LEMI92 — COMMENTED** [review]
{
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:22791",
      "sslPort": 44390
    }
  },
  "profiles": {
    "Modelo": {
      "commandName": "Project",
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7049,http://localhost:5208",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}

File: launchSettings.json
Comment (KarinaVillacorta21) [inline launchSettings.json:1]: {
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:22791",
      "sslPort": 44390
    }
  },
  "profiles": {
    "Modelo": {
      "commandName": "Project",
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7049,http://localhost:5208",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}

**Review by LEMI92 — COMMENTED** [review]
{
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:22791",
      "sslPort": 44390
    }
  },
  "profiles": {
    "Modelo": {
      "commandName": "Project",
      "launchBrowser": true,
      "applicationUrl": "https://localhost:7049,http://localhost:5208",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}

File: launchSettings.json
Comment (LEMI92) [inline launchSettings.json:1]: correccion de error ; por ,

**Pull request merged**